src/main/java/kostka/moviecatalog/entity/counters/SynchronizedCounter.java

```diff
@@ -0,0 +1,25 @@
+package kostka.moviecatalog.entity.counters;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table
+public class SynchronizedCounter {
+    @Id
+    private Long id;
+    volatile int counter;
+
+    public synchronized void syncedIncrement() {
+        counter++;
+    }
+
+    public int getCounterValue() {
+        return counter;
+    }
+
+    public void setCounterValue(final int counter) {
```

Comment:
Tohle taky pristupuje k citaci, tak by to taky mohlo byt jako synchronized.

Reply:
Zatím jsem tuhle. Metodu ještě nepoužil, ale máš pravdu, taky vy to mělo být synchronized